apps/docs/content/guides/self-hosting/docker.mdx

```diff
@@ -67,6 +67,9 @@ docker compose up -d
 </TabPanel>
 </Tabs>
 
+If you are using rootless docker, edit `.env` and set `DOCKER_SOCKET_LOCATION` to your docker socket location. For example: `/run/user/1000/docker.sock`.
+Or you will see an error like `container supabase-vector exited (0)`.
```

Comment:
```suggestion
If you are using rootless docker, edit `.env` and set `DOCKER_SOCKET_LOCATION` to your docker socket location. For example: `/run/user/1000/docker.sock`. Otherwise, you will see an error like `container supabase-vector exited (0)`.
```